signer_client.go

```diff
@@ -493,21 +495,34 @@ func MuSig2TaprootTweakOpt(scriptRoot []byte,
 // MuSig2CreateSession creates a new musig session with the key and signers
 // provided.
 func (s *signerClient) MuSig2CreateSession(ctx context.Context,
-	signerLoc *keychain.KeyLocator, signers [][32]byte,
-	opts ...MuSig2SessionOpts) (*input.MuSig2SessionInfo, error) {
+	version input.MuSig2Version, signerLoc *keychain.KeyLocator,
+	signers [][]byte, opts ...MuSig2SessionOpts) (
+	*input.MuSig2SessionInfo, error) {
 
-	signerBytes := make([][]byte, len(signers))
-	for i, signer := range signers {
-		signerBytes[i] = make([]byte, 32)
-		copy(signerBytes[i], signer[:])
+	var rpcMuSig2Version signrpc.MuSig2Version
+
+	// Select the version based on the passed Go enum. Note that with new
```

Comment:
nit: we could add a helper func to marshal/unmarshal musig versions

Reply:
done